tests/integration/test_fim/test_files/test_follow_symbolic_link/test_audit_rules_removed_after_change_link.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+import os
+import subprocess
+
+import pytest
+
+
+from wazuh_testing.fim import generate_params, create_file, REGULAR, SYMLINK, callback_detect_event, \
+                              LOG_FILE_PATH, change_internal_options
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing import global_parameters
+
+from test_fim.test_files.test_follow_symbolic_link.common import configurations_path, testdir1, \
+    modify_symlink, testdir_not_target, test_directories, wait_for_audit
+
+pytestmark = [pytest.mark.linux, pytest.mark.tier(level=1)]
+
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+# Variables
+
+fname = "testfile"
+symlink_root_path = "/"
```

Comment:
This variable is similar to `PREFIX` defined in the testing tools, but is not portable to other systems, please replace it.